Lavanya/HTMLassignment3.html

```diff
@@ -1,31 +1,40 @@
 <html>
-
+<head>
+	<title>HTMLassignment3</title>
+	<meta charset="UTF-8">
+</head>
 <body>
 
 	<h2>Register</h2>
 
-	<form>
+	<form id="RegistrationForm">
 		<label for="First Name">First Name</label>
 		<input type="text" name="First Name"required/>
 		<label for="Last Name">Last Name</label>
 		<input type="text" name="lastname"required>
 		<br><br>
-		<label for="gender">Gender:</label>
-		<input type="radio" name="gender" value="male" checked required/> Male
-		<input type="radio" name="gender" value="female"> Female
-		<input type="radio" name="gender" value="others"> Others
+		Gender:
+		<label> Male 
+                	<input type="radio" name="gender" id="male" value="male" checked required/> 
+		</label>
+   		<label> Female
+			<input type="radio" name="gender" id="female" value="female">
+		</label> 
+		<label> Others
+			<input type="radio" name="gender" id="others" value="others">
+		</label>

```

Comment:
This is wrong syntax. Previous was right. And male shouldn't be checked by default as it can end up in accidental submission and also offensive to other genders in certain cases.